openedx/core/djangoapps/performance/tests/test_logs.py

```diff
@@ -1,4 +1,7 @@
-"""Tests that performance data is successfully logged."""
+# pylint: disable=no-member
```

Comment:
There were many lines that needed it.  And for test code, we generally go ahead and quiet pylint for the issues such as `no-member`.